plugin/nulls.go

```diff
@@ -0,0 +1,74 @@
+package plugin
+
+import (
+	"github.com/apache/arrow/go/v13/arrow"
+	"github.com/apache/arrow/go/v13/arrow/array"
+	"github.com/apache/arrow/go/v13/arrow/memory"
+)
+
+//nolint:unused
+func stripNullsFromLists(records []arrow.Record) {
```

Comment:
Why are we adding unused code? Will there be a follow-up where it gets used?

Reply:
No, I was just migrating CH when I noticed this missing.

I can comment out the relevant section in CH tests, but it will still be required once we bring the vast testing back

Reply:
We can leave this here as a reminder, but can we rather add it back when it's needed?

Reply:
💯